.github/workflows/scripts/copyright_excludes.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright check exclude patterns
```

Comment:
Could you please comment on why we are excluding these folders? Are these mostly third party folders with non-Apache 2.0 licenses?

Reply:
Yes, 3rd party copyright headers in these folders